### PR TITLE
Reorder the admin nav items into alphabetical order

### PR DESCRIPTION
**Pull request by settermjd**
The intent here is to make the configuration -> server section easier to scan for users, and ultimately quicker and more efficient to use.

### CODE REVIEW

**Comment by settermjd** [comment]
Thanks, @pako81.

**Comment by pako81** [comment]
you are welcome @settermjd

File: modules/admin_manual/nav.adoc [hunk 1]
```diff
@@ -97,16 +91,22 @@
 **** xref:configuration/server/harden_server.adoc[Harden Server]
 **** xref:configuration/server/import_ssl_cert.adoc[Import SSL Cert]
 **** xref:configuration/server/index_php_less_urls.adoc[Index PHP Less URLs]
+**** xref:configuration/server/occ_command.adoc[OCC Command]
```
Comment (mmattel) [inline modules/admin_manual/nav.adoc]: why did you put occ_command and reverse_proxy out of the alphabetical sort order of the `****` list level?

Comment (pako81) [reply modules/admin_manual/nav.adoc]: occ_command is correct. Yes,  reverse_proxy_configuration is wrong indeed. Sorry I missed it.

**Pull request merged**